provider/cloudflare/cloudflare.go

```diff
@@ -150,6 +159,14 @@ func getUpdateDNSRecordParam(cfc cloudFlareChange) cloudflare.UpdateDNSRecordPar
 	}
 }
 
+// getUpdateDataLocalizationRegionalHostnameParams is a function that returns the appropriate RegionalHostname Param based on the cloudFlareChange passed in
+func getUpdateDataLocalizationRegionalHostnameParams(cfc cloudFlareChange) cloudflare.UpdateDataLocalizationRegionalHostnameParams {
+	return cloudflare.UpdateDataLocalizationRegionalHostnameParams{
+		Hostname:  cfc.RegionalHostname.Hostname,
+		RegionKey: cfc.RegionalHostname.RegionKey,
+	}
+}
+
```

Comment:
A `getSomething` should _never_ `updateSomething`.
You should either rename this method or rework the content of it.

Reply:
I based the function name on the one declared directly above, `getUpdateDNSRecordParam`. Should that be renamed as well?

Reply:
I updated my function names. I imagine a separate MR is needed to change @mloiseleur's naming